addon/client.go

```diff
@@ -166,7 +170,8 @@ func (r *Client) Put(path string, object interface{}) (err error) {
 	status := reply.StatusCode
 	switch status {
 	case http.StatusNoContent:
-	case http.StatusOK:
+	case http.StatusOK,
+		http.StatusCreated:
```

Comment:
**hint**: By definition PUT can create resources.  Added for completeness even though the API currently does not have any endpoints that do this.